pkg/scheduler/framework/plugins/nodevolumelimits/csi.go

```diff
@@ -232,13 +232,13 @@ func (pl *CSILimits) getCSIDriverInfoFromSC(csiNode *storagev1.CSINode, pvc *v1.
 	// If StorageClass is not set or not found, then PVC must be using immediate binding mode
 	// and hence it must be bound before scheduling. So it is safe to not count it.
 	if scName == "" {
-		klog.V(5).Infof("PVC %s/%s has no StorageClass", namespace, pvcName)
+		klog.V(5).InfoS("PVC has no StorageClass", "PVC", klog.KRef(namespace, pvcName))
 		return "", ""
 	}
 
 	storageClass, err := pl.scLister.Get(scName)
 	if err != nil {
-		klog.V(5).Infof("Could not get StorageClass for PVC %s/%s: %v", namespace, pvcName, err)
+		klog.V(5).ErrorS(err, "Could not get StorageClass for PVC", "PVC", klog.KRef(namespace, pvcName))
```

Comment:
According to line 232, it is fine to leave this as info.

Reply:
Thanks, done.